include/debug/thread_analyzer.h

```diff
@@ -32,6 +32,10 @@ struct thread_analyzer_info {
 
 #ifdef CONFIG_THREAD_RUNTIME_STATS
 	unsigned int utilization;
+	uint64_t  current_cycles;
```

Comment:
It is a good observation, and I'm was not entirely happy with the multiple definitions. Similar data exists in struct thread_analyzer_info, struct k_thread_runtime_stats, struct _thread_base and struct _cpu. The first two cases are more output/user oriented (slightly massaged data), and the latter two more internally oriented (raw data).

If desired, I am quite happy to re-jigger it so that there are two structures--one internally oriented (raw data), and one output/user oriented.